addons/html_editor/static/src/main/link/link_plugin.js

```diff
@@ -160,7 +188,10 @@ export class LinkPlugin extends Plugin {
         handle_insert_line_break_element: { callback: p.handleInsertLineBreak.bind(p) },
     });
     setup() {
-        this.overlay = this.shared.createOverlay(LinkPopover);
+        this.overlay = this.shared.createOverlay(LinkPopover, {
+            position: "bottom-start",
+            sequence: 40,
+        });
```

Comment:
Why ? bottom-start is the default ?
Why do you need sequence ?

Reply:
I used the sequence to make sure when we select the image link and scroll the page, if the toolbar is repositioned to the bottom of the image and overlaps the link popover, the link popover is behind the toolbar